aws/resource_aws_ecs_task_set_test.go

```diff
@@ -0,0 +1,1076 @@
+package aws
```

Comment:
can you switch the order of the tests and test configs so that the test themselves are on top and configs are at the bottom? makes it more readable and aligned with other resource tests